tests/stress/conftest.py

```diff
@@ -34,9 +34,19 @@ def set_polling_interval(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     time.sleep(wait_time)
 
 
+@pytest.fixture(scope="module")
+def cleanup_neighbors_dualtor(duthosts, ptfhost, tbinfo):
+    """Cleanup neighbors on dualtor testbed."""
+    if "dualtor" in tbinfo["topo"]["name"]:
+        ptfhost.shell("supervisorctl stop garp_service", module_ignore_errors=True)
+        ptfhost.shell("supervisorctl stop arp_responder", module_ignore_errors=True)
+        duthosts.shell("sonic-clear arp")
+        duthosts.shell("sonic-clear ndp")
+
+
 @pytest.fixture(scope='module')
 def withdraw_and_announce_existing_routes(duthosts, localhost, tbinfo, enum_rand_one_per_hwsku_frontend_hostname,
-                                          enum_rand_one_frontend_asic_index):
+                                          enum_rand_one_frontend_asic_index, cleanup_neighbors_dualtor):            # noqa F811
```

Comment:
IMO, `withdraw_and_announce_existing_routes` can cleanup all routes. Why do you add another fixture to clean?

Reply:
On dualtor testbeds, tunnel routes are counted in the crm IPv4/IPv6 route counters:
https://github.com/sonic-net/sonic-swss/blob/29cea04a0b98cc440fd5730c758a9b73ac95362c/orchagent/muxorch.cpp#L129-L138